tests/data/config/t2sa_test.yaml

```diff
@@ -1,4 +1,4 @@
-t2sa_ip: "192.168.0.84"
+t2sa_ip: "140.252.33.47"
```

Comment:
It seems a bit odd that you need to have these IPs in a test configuration..

Reply:
It's because these tests need to talk to the real T2SA server running on a windows host with a SpatialAnalyzer license, The T2SA application has its own simulation mode that allows it to accept API commands and simulate responses without there actually being a laser tracker connected.